docs/source/conf.py

```diff
@@ -0,0 +1,159 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+from pkg_resources import get_distribution
+
+from m2r import MdInclude
+from recommonmark.transform import AutoStructify
+
+# -- Project information -----------------------------------------------------
+
+project = 'labscript-utils'
+copyright = '2020, labscript suite'
+author = 'labscript suite contributors'
+
+
+# The full version, including alpha/beta/rc tags
+version = get_distribution(project).version
```

Comment:
Shouldn't this be:
``python
from labscript_suite.__version__ import __version__
```

Reply:
do you mean `from labscript_utils ...`? I also don't know - I copied it from your lyse repo [here](https://github.com/rpanderson/lyse/blob/docs-sphinx/docs/source/conf.py#L27)

Reply:
> do you mean `from labscript_utils ...`? 

I absolutely do!

> I copied it from your lyse repo [here](https://github.com/rpanderson/lyse/blob/docs-sphinx/docs/source/conf.py#L27)

Ah, yep. That was written before we had converged on the current versioning method, which I hope will remain static for a while now. My proposed change is working in the labscript-suite metapackage docs.

Reply:
This is now updated